src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private static final Truck first = new Truck();
+    private static final Truck second = new Truck();
+    private static final Truck third = new Truck();
+
+    @Override
+    public List<Truck> get() {
+        return List.of(first, second, third);
+    }
+}
```

Comment:
You create three same equal constants,  i think it's redundant. You can try this solution.
```suggestion
    private static final int SIZE_OF_LIST = 3;

    @Override
    public List<Truck> get() {
        List<Truck> list = new ArrayList<>();
        for (int i = 0; i < SIZE_OF_LIST; i++) {
             list.add(new Truck());
        }
        return list;
    }
}
```

Reply:
I dont think its need this because it is template, a was lazy to add some fields in those classes, and if i did it i couldn't use cycle))